docs/tctl.md

```diff
@@ -269,9 +269,10 @@ You can reset to some predefined event types:
 ./tctl workflow reset -w <wid> -r <rid> --reset_type <reset_type> --reason "some_reason"
 ```
 
-- FirstDecisionCompleted: reset to the beginning of the history.
-- LastDecisionCompleted: reset to the end of the history.
+- FirstWorkflowTask: reset to the beginning of the history.
+- LastWorkflowTask: reset to the end of the history.
 - LastContinuedAsNew: reset to the end of the history for the previous run.
+- BadBinary: <>.
```

Comment:
```suggestion
- BadBinary: reset to the point where the version of the binary would throw.
```

Reply:
@cullywakelin does this wording look fine?

Reply:
Maybe just:
> Reset to the point where a bad binary was used.